ui/src/app/shared/pipes/duration.pipe.ts

```diff
@@ -6,13 +6,15 @@ import {Pipe, PipeTransform} from '@angular/core';
 export class DurationPipe implements PipeTransform {
 
   transform(start: Date, end: Date): string {
-    let duration: number;
-    if (end) {
-      duration = end.getTime() - start.getTime();
-    } else {
-      duration = new Date().getTime() - start.getTime();
-    }
-    return Math.floor(duration/3600000) + "h " +
-      Math.floor(duration/60000)%60 + "m";
+   if (start) {
```

Comment:
If I understand it correctly, there are indeed some situations that `start` is missing?

Reply:
There were, but I believe I've addressed them, so I've removed this check – thanks for the catch.